### PR TITLE
HTML-Based GUI Configuration Utility

### PR DESCRIPTION
**Pull request by davidseibel**
This includes a new HTML-based configuration utility created during the VMware Code Hackathon at VMworld.  To invoke this new interface, run .\vCheck.ps1 -GUIConfig.  

The existing Import-vCheckSettings tool does not work well in CSV format due to the presence of commas in questions.  Because of this, an XML version of the Import-vCheckSettings (Import-vCheckSettingsXML) utility function was created to support imports from the HTML config tool (an equivalent Export-vCheckSettingsXML function was also created).

### CODE REVIEW

**Comment by alanrenouf** [comment]
This is super awesome, so impressed with this and how fast you created this elegant feature in the time we were given. Fantastic. Will download and test soon.

**Comment by Sneddo** [comment]
Agreed, looking forward to trying this out tomorrow.

File: vCheck.ps1 [hunk 1]
```diff
@@ -58,6 +58,8 @@
 param (
 	[Switch]$config,
 
+    [Switch]$GUIConfig,
```
Comment (rnelson0) [inline vCheck.ps1]: Indent level

File: vCheck.ps1 [hunk 2]
```diff
@@ -239,6 +244,79 @@ Function Invoke-Settings {
 	
 } # end Function Invoke-Settings
 
+Function Invoke-HTMLSettings {
+	
+	<#
+	.DESCRIPTION
+		Run through settings for specified file, expects question on one line, and variable/value on following line.
```
Comment (rnelson0) [inline vCheck.ps1]: Indents in this block as well.

**Comment by rnelson0** [comment]
This is pretty awesome, it's very simple, much better than re-running the interactive config to change one value. This looks good if the indenting is fixed, would make it much easier to adjust later.

**Comment by davidseibel** [comment]
It should be fixed now.  Was using spaces on those lines instead of tabs.

Thanks for the feedback!

**Comment by rnelson0** [comment]
I'm not sure which is preferred, spaces or tabs, but this looks much better and causes everything to line up. :+1: from me!

**Pull request merged**